server/pkg/scene/style_value.go

```diff
@@ -0,0 +1,3 @@
+package scene
+
+type StyleValue map[string]struct{}
```

Comment:
```suggestion
type StyleValue map[string]any
```

Reply:
I'm not sure, if the value can not be a primitive type plz ignore this comment

Reply:
It can be that, resolved.